packages/server/src/ui/routes/project-dashboard/graphs/category-score/category-score-graph.jsx

```diff
@@ -56,6 +64,7 @@ const HoverCardWithDiff = props => {
     children = (
       <Fragment>
         {versionChange ? <VersionChangeWarning /> : null}
+        {hideInfoTip || props.pinned ? null : <InfoTip />}
```

Comment:
oh yeh good catch! I had this thought and then abandoned it for this first pass.